CloudFoundry/content/getting-started/concepts.md

```diff
@@ -75,10 +75,6 @@ cf target -o ORGNAME -s SPACENAME
 All apps need to use a 'buildpack' specific to their language, which sets up dependencies for particular language stacks. There are standard buildpacks for most lanugages, and they will usually be auto-detected by CF. Using the standard buildpacks is strongly encouraged. In the rare case where the buildpack does not get detected correctly, or to use a custom buildpack, it can be specified in the manifest (as below) or with the `-b` flag. Use either the buildpack name:
```

Comment:
The phrase `Use either the buildpack name` no longer is quite correct now that there's only the one option below.